Header header
scene_graph/RoomPolygon[] rooms